include/rddlSDKAPI.h

```diff
@@ -1,25 +1,36 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
+#define SDK_STACK_LIMIT 7168
+
 
 const char* sdkGetRDDLAddress();
 const char* sdkGetExtPubKeyLiquid();
 const char* sdkGetExtPubKeyPlanetmint();
 const uint8_t* sdkGetPrivKeyLiquid();
 const uint8_t* sdkGetPrivKeyPlanetmint();
 const char* sdkGetMachinePublicKey();
+
 bool  sdkGetPlntmntKeys();
 char* sdkSetSeed(char* pMnemonic, size_t len);
+
 void sdkStoreSeed(char* new_seed);
 uint8_t* sdkReadSeed(char* seed_arr, int* seed_size);
+
 int sdkReadFile( const char* filename, uint8_t* content, size_t length);
 bool sdkGetAccountInfo( uint64_t* account_id, uint64_t* sequence );
+
 char* sdkGetSetting(uint32_t index);
 bool sdkSetSetting(uint32_t index, const char* replacementText);
 
+uint8_t* sdkGetStack( size_t size );
```

Comment:
The SDK interface exists to make librddl easier to use. It does not need to provide memory to the user. Is there a need for sdkGetStack and sdkClearStack?

Reply:
the getstack is needed on tasmota. it would not work otherwise. we can change that asap we migrated librddl and rddl-sdk so that we can use the stack explicitly without the need to reflect it via the sdk methods.

we currently take around 7kb for the stack. and I want to share this with some application logic on tasmota. we might move some of the logic to the SDK. having that we are able to remove the stack from the exposure.